examples/package.json

```
@@ -9,10 +9,10 @@
     "decodeData": "node src/decodeData.js"
   },
   "dependencies": {
-    "@erc725/erc725.js": "^0.17.2",
+    "@erc725/erc725.js": "..",
```

maybe we can just put this?

```suggestion
    "@erc725/erc725.js": "latest",
```

---

Ok we could do that, I have .. so it installs the currently compiled code.